encfs/SSL_Cipher.cpp

```diff
@@ -153,6 +153,47 @@ int TimedPBKDF2(const char *pass, int passlen, const unsigned char *salt,
 // - Version 3:0 adds a new IV mechanism
 static Interface BlowfishInterface("ssl/blowfish", 3, 0, 2);
 static Interface AESInterface("ssl/aes", 3, 0, 2);
+static Interface CAMELLIAInterface("ssl/camellia",3, 0, 2);
+
+#ifndef OPENSSL_NO_CAMELLIA
+
+static Range CAMELLIAKeyRange(128, 256, 32);
```

Comment:
change 32 -> 64.  Key lengths other than 128, 192, and 256 shouldn't be provided.  Looks like any other length is translated to 256bit in the switch below.